feature1/oneMoreFile.txt

```diff
@@ -0,0 +1,3 @@
+This feature was also added locally from feature1 branch
```

Comment:
Its really good that you have decided to add this line

Reply:
Thanks for this comment, lone7